src/app/core/layouts/navigation-bar/navigation-bar.component.ts

```diff
@@ -1,15 +1,21 @@
 import { Component, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { NewTutorialDialogComponent } from 'src/app/features/new-tutorial-dialog/new-tutorial-dialog.component';
 
 @Component({
   selector: 'app-navigation-bar',
   templateUrl: './navigation-bar.component.html',
   styleUrls: ['./navigation-bar.component.less']
 })
 export class NavigationBarComponent implements OnInit {
+  constructor(public dialog: MatDialog) {}
 
-  constructor() { }
-
-  ngOnInit() {
+  ngOnInit() {}
+  openDialog() {
+    console.log('open sesemy');
```

Comment:
should we leave console logs in the code for now?

Reply:
no i will fix it, 
we should only do errors in the mean time